hawkbit-dmf-amqp/src/main/java/org/eclipse/hawkbit/amqp/AmqpConfiguration.java

```diff
@@ -58,6 +60,9 @@
     @Autowired
     private ConnectionFactory rabbitConnectionFactory;
 
+    @Autowired
+    private ErrorHandler errorHandler;
```

Comment:
Are you really need this variable? Just call

`return new ConfigurableRabbitListenerContainerFactory(amqpProperties, rabbitConnectionFactory, errorHandler());`